takeshape/penny
# clean up, reorg

## Test step

1. CI...
2. Go to the preview site, click around, any issues should be immediately apparent.
3. Pull the branch, make sure `npm run storybook` is working correctly
